book/src/03_ticket_v1/07_setters.md

```diff
@@ -50,7 +50,7 @@ It takes ownership of `self`, changes the title, and returns the modified `Ticke
 This is how you'd use it:
 
 ```rust
-let ticket = Ticket::new("Title".into(), "Description".into(), "To-Do".into());
+let mut ticket = Ticket::new("Title".into(), "Description".into(), "To-Do".into());
```

Comment:
This one isn't necessary even if the function argument were to be changed to mut because you are passing ownership so the function is allowed to modify it if it chooses.